digital-land/digital-land-python
# Entity organisation owen

## What type of PR is this? (check all applicable)


## Description

This pull request aims to introduce a priority phase into the pipeline, specifically to allow the DM team to assign an organisation to each entity which receives priority over any other organisation.

## Related Tickets & Documents


- [Ticket Link](https://trello.com/c/h3W7cGVx)

## QA Instructions, Screenshots, Recordings

There is a branch of the conservation-area-collection repo which contains an entity-organisation.csv file. Running the code on this branch (combined with the changes to the [makerules repo](https://github.com/digital-land/makerules/pull/44)) should show an example of how this is used in practice. Specifically that a config.sqlte3 is made and in the datasets at. the end there should be some values with a higher priority.

Also choose another repo without the entity-organisation.csv in it and run the new code. It has to be able to function when entity-organisation is not provided.

Full process:

* checkout conservation-area-collection repo
* Update makerules from https://github.com/digital-land/makerules/pull/44 (done by. manuallly copying or changing pipelie makerules url)
* get config files from https://github.com/digital-land/config/pull/189 (done be changin the config url to CONFIG_URL=https://raw.githubusercontent.com/digital-land/config/feat/entity-organisation/)
* run make -j 10
* Check datasette using make datasette.  you should seen priority has been assigned in the fact resource and fact tables. Some values should be 1 for conservation-area
* check that config.sqlite3 has been made and is non empty for conservation area

repeat the above for another collection but there will be no rows with priority 1 and the config.sqlite3 entity-organisation table will be empty 


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] Are there any dependencies on other PRs or Work?
the makerules PR found here will need to be merged at the same time
